generate-versions.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [ $# != 2 ]; then
+    echo "Usage: $0 <output file> <extra tag>" 1>&2
+    exit 1
+fi
+
+VERSIONS="$1"
+EXTRA="$2"
+
+DIRS="testpmd-container-app trex-container-app cnf-app-mac-operator testpmd-lb-operator testpmd-operator trex-operator nfv-example-cnf-index"
+
+FILES="$(git diff --name-only -r main)"
```

Comment:
The output of the job is saying:
fatal: ambiguous argument 'main': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
which is weird, it works properly on my laptop...

Reply:
this was because the git history was not compete.